sdk/core/core-http/lib/operationOptions.ts

```diff
@@ -0,0 +1,64 @@
+import { AbortSignalLike } from "@azure/abort-controller";
+import { SpanOptions } from "@azure/core-tracing";
+import { TransferProgressEvent, RequestOptionsBase } from "./webResource";
+
+/**
+ * The base options type for all operations.
+ */
+export interface OperationOptions {
+  /**
+   * The signal which can be used to abort requests.
+   */
+  abortSignal?: AbortSignalLike;
```

Comment:
nit: Should there be spacing between the fields and comment blocks like in OperationRequestOptions below?